modules/anki/testbeds/anki.nix

```
@@ -0,0 +1,11 @@
+{ lib, ... }:
+{
+  stylix.testbed.ui.command.text = "anki";
+
+  home-manager.sharedModules = lib.singleton {
+    programs.anki = {
+      enable = true;
+      language = "eo_UY";
+    };
```

Keep the testbed generic:

```suggestion
    programs.anki.enable = true;
```